2023-07-29 07:04:02,304 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:04:02,304 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:04:02,304 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:04:02,305 - utils - INFO - Started main() method...
2023-07-29 07:04:02,305 - utils - INFO - Started main() method...
2023-07-29 07:04:02,305 - utils - INFO - Started main() method...
2023-07-29 07:04:02,305 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:04:02,305 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:04:02,305 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:04:02,305 - utils - INFO - Running on dev environment
2023-07-29 07:04:02,305 - utils - INFO - Running on dev environment
2023-07-29 07:04:02,305 - utils - INFO - Running on dev environment
2023-07-29 07:04:04,356 - utils - INFO - Spark object created..
2023-07-29 07:04:04,356 - utils - INFO - Spark object created..
2023-07-29 07:04:04,356 - utils - INFO - Spark object created..
2023-07-29 07:04:04,356 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:04:04,356 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:04:04,356 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:04:04,356 - utils - WARNING - Validation done..!
2023-07-29 07:04:04,356 - utils - WARNING - Validation done..!
2023-07-29 07:04:04,356 - utils - WARNING - Validation done..!
2023-07-29 07:14:15,257 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:14:15,257 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:14:15,257 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:14:15,258 - utils - INFO - Started main() method...
2023-07-29 07:14:15,258 - utils - INFO - Started main() method...
2023-07-29 07:14:15,258 - utils - INFO - Started main() method...
2023-07-29 07:14:15,258 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:14:15,258 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:14:15,258 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:14:15,258 - utils - INFO - Running on dev environment
2023-07-29 07:14:15,258 - utils - INFO - Running on dev environment
2023-07-29 07:14:15,258 - utils - INFO - Running on dev environment
2023-07-29 07:14:17,239 - utils - INFO - Spark object created..
2023-07-29 07:14:17,239 - utils - INFO - Spark object created..
2023-07-29 07:14:17,239 - utils - INFO - Spark object created..
2023-07-29 07:14:17,239 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:14:17,239 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:14:17,239 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:14:17,239 - utils - WARNING - Validation done..!
2023-07-29 07:14:17,239 - utils - WARNING - Validation done..!
2023-07-29 07:14:17,239 - utils - WARNING - Validation done..!
2023-07-29 07:15:52,068 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:15:52,068 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:15:52,068 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:15:52,069 - utils - INFO - Started main() method...
2023-07-29 07:15:52,069 - utils - INFO - Started main() method...
2023-07-29 07:15:52,069 - utils - INFO - Started main() method...
2023-07-29 07:15:52,069 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:15:52,069 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:15:52,069 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:15:52,069 - utils - INFO - Running on dev environment
2023-07-29 07:15:52,069 - utils - INFO - Running on dev environment
2023-07-29 07:15:52,069 - utils - INFO - Running on dev environment
2023-07-29 07:15:54,102 - utils - INFO - Spark object created..
2023-07-29 07:15:54,102 - utils - INFO - Spark object created..
2023-07-29 07:15:54,102 - utils - INFO - Spark object created..
2023-07-29 07:15:54,102 - utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 142, in main
    if test_spark_obj(spark):
TypeError: test_spark_obj() missing 1 required positional argument: 'logger'
2023-07-29 07:15:54,102 - utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 142, in main
    if test_spark_obj(spark):
TypeError: test_spark_obj() missing 1 required positional argument: 'logger'
2023-07-29 07:15:54,102 - utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 142, in main
    if test_spark_obj(spark):
TypeError: test_spark_obj() missing 1 required positional argument: 'logger'
2023-07-29 07:17:08,585 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:17:08,585 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:17:08,585 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:17:08,586 - utils - INFO - Started main() method...
2023-07-29 07:17:08,586 - utils - INFO - Started main() method...
2023-07-29 07:17:08,586 - utils - INFO - Started main() method...
2023-07-29 07:17:08,586 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:17:08,586 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:17:08,586 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:17:08,586 - utils - INFO - Running on dev environment
2023-07-29 07:17:08,586 - utils - INFO - Running on dev environment
2023-07-29 07:17:08,586 - utils - INFO - Running on dev environment
2023-07-29 07:17:10,657 - utils - INFO - Spark object created..
2023-07-29 07:17:10,657 - utils - INFO - Spark object created..
2023-07-29 07:17:10,657 - utils - INFO - Spark object created..
2023-07-29 07:17:10,657 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:17:10,657 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:17:10,657 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:17:10,657 - utils - WARNING - Validation done..!
2023-07-29 07:17:10,657 - utils - WARNING - Validation done..!
2023-07-29 07:17:10,657 - utils - WARNING - Validation done..!
2023-07-29 07:19:12,008 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:19:12,008 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:19:12,008 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:19:12,008 - utils - INFO - Started main() method...
2023-07-29 07:19:12,008 - utils - INFO - Started main() method...
2023-07-29 07:19:12,008 - utils - INFO - Started main() method...
2023-07-29 07:19:12,008 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:19:12,008 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:19:12,008 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:19:12,008 - utils - INFO - Running on dev environment
2023-07-29 07:19:12,008 - utils - INFO - Running on dev environment
2023-07-29 07:19:12,008 - utils - INFO - Running on dev environment
2023-07-29 07:19:13,984 - utils - INFO - Spark object created..
2023-07-29 07:19:13,984 - utils - INFO - Spark object created..
2023-07-29 07:19:13,984 - utils - INFO - Spark object created..
2023-07-29 07:19:13,984 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:19:13,984 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:19:13,984 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:19:13,984 - utils - WARNING - Validation done..!
2023-07-29 07:19:13,984 - utils - WARNING - Validation done..!
2023-07-29 07:19:13,984 - utils - WARNING - Validation done..!
2023-07-29 07:21:44,102 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:21:44,102 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:21:44,102 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:21:44,102 - utils - INFO - Started main() method...
2023-07-29 07:21:44,102 - utils - INFO - Started main() method...
2023-07-29 07:21:44,102 - utils - INFO - Started main() method...
2023-07-29 07:21:44,102 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:21:44,102 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:21:44,102 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:21:44,103 - utils - INFO - Running on dev environment
2023-07-29 07:21:44,103 - utils - INFO - Running on dev environment
2023-07-29 07:21:44,103 - utils - INFO - Running on dev environment
2023-07-29 07:21:46,035 - utils - INFO - Spark object created..
2023-07-29 07:21:46,035 - utils - INFO - Spark object created..
2023-07-29 07:21:46,035 - utils - INFO - Spark object created..
2023-07-29 07:21:46,035 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:21:46,035 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:21:46,035 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:21:46,035 - utils - WARNING - Validation done..!
2023-07-29 07:21:46,035 - utils - WARNING - Validation done..!
2023-07-29 07:21:46,035 - utils - WARNING - Validation done..!
2023-07-29 07:32:03,084 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:32:03,084 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:32:03,084 - utils - INFO - logger object is created and file location is local dir
2023-07-29 07:32:03,084 - utils - INFO - Started main() method...
2023-07-29 07:32:03,084 - utils - INFO - Started main() method...
2023-07-29 07:32:03,084 - utils - INFO - Started main() method...
2023-07-29 07:32:03,085 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:32:03,085 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:32:03,085 - utils - INFO - Started get_spark_obj() method..
2023-07-29 07:32:03,086 - utils - INFO - Running on dev environment
2023-07-29 07:32:03,086 - utils - INFO - Running on dev environment
2023-07-29 07:32:03,086 - utils - INFO - Running on dev environment
2023-07-29 07:32:05,107 - utils - INFO - Spark object created..
2023-07-29 07:32:05,107 - utils - INFO - Spark object created..
2023-07-29 07:32:05,107 - utils - INFO - Spark object created..
2023-07-29 07:32:05,107 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:32:05,107 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:32:05,107 - utils - WARNING - Started the test_spark_obj method
2023-07-29 07:32:05,107 - utils - WARNING - Validation done..!
2023-07-29 07:32:05,107 - utils - WARNING - Validation done..!
2023-07-29 07:32:05,107 - utils - WARNING - Validation done..!
2023-07-29 08:33:47,932 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:33:47,932 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:33:47,932 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:33:47,932 - log_utils - INFO - Started main() method...
2023-07-29 08:33:47,932 - log_utils - INFO - Started main() method...
2023-07-29 08:33:47,932 - log_utils - INFO - Started main() method...
2023-07-29 08:33:47,932 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 37, in main
    spark = get_spark_obj(envn, spark_app_name, logger)
  File "/home/ubuntu/aws_restbus_proj/source/create_spark.py", line 10, in get_spark_obj
    config = load_config(config_path, 'r')
TypeError: load_config() missing 1 required positional argument: 'logger'
2023-07-29 08:33:47,932 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 37, in main
    spark = get_spark_obj(envn, spark_app_name, logger)
  File "/home/ubuntu/aws_restbus_proj/source/create_spark.py", line 10, in get_spark_obj
    config = load_config(config_path, 'r')
TypeError: load_config() missing 1 required positional argument: 'logger'
2023-07-29 08:33:47,932 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 37, in main
    spark = get_spark_obj(envn, spark_app_name, logger)
  File "/home/ubuntu/aws_restbus_proj/source/create_spark.py", line 10, in get_spark_obj
    config = load_config(config_path, 'r')
TypeError: load_config() missing 1 required positional argument: 'logger'
2023-07-29 08:35:09,504 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:35:09,504 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:35:09,504 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:35:09,504 - log_utils - INFO - Started main() method...
2023-07-29 08:35:09,504 - log_utils - INFO - Started main() method...
2023-07-29 08:35:09,504 - log_utils - INFO - Started main() method...
2023-07-29 08:35:09,505 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:35:09,505 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:35:09,505 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:35:09,506 - log_utils - INFO - Running on dev environment
2023-07-29 08:35:09,506 - log_utils - INFO - Running on dev environment
2023-07-29 08:35:09,506 - log_utils - INFO - Running on dev environment
2023-07-29 08:35:11,555 - log_utils - INFO - Spark object created..
2023-07-29 08:35:11,555 - log_utils - INFO - Spark object created..
2023-07-29 08:35:11,555 - log_utils - INFO - Spark object created..
2023-07-29 08:35:11,555 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:35:11,555 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:35:11,555 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:35:11,555 - log_utils - WARNING - Validation done..!
2023-07-29 08:35:11,555 - log_utils - WARNING - Validation done..!
2023-07-29 08:35:11,555 - log_utils - WARNING - Validation done..!
2023-07-29 08:35:11,555 - log_utils - INFO - Application done!!!
2023-07-29 08:35:11,555 - log_utils - INFO - Application done!!!
2023-07-29 08:35:11,555 - log_utils - INFO - Application done!!!
2023-07-29 08:38:31,593 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:38:31,593 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:38:31,593 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:38:31,593 - log_utils - INFO - Started main() method...
2023-07-29 08:38:31,593 - log_utils - INFO - Started main() method...
2023-07-29 08:38:31,593 - log_utils - INFO - Started main() method...
2023-07-29 08:38:31,594 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:38:31,594 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:38:31,594 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:38:31,594 - log_utils - INFO - Running on dev environment
2023-07-29 08:38:31,594 - log_utils - INFO - Running on dev environment
2023-07-29 08:38:31,594 - log_utils - INFO - Running on dev environment
2023-07-29 08:38:33,612 - log_utils - INFO - Spark object created..
2023-07-29 08:38:33,612 - log_utils - INFO - Spark object created..
2023-07-29 08:38:33,612 - log_utils - INFO - Spark object created..
2023-07-29 08:38:33,612 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:38:33,612 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:38:33,612 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:38:33,612 - log_utils - WARNING - Validation successful!
2023-07-29 08:38:33,612 - log_utils - WARNING - Validation successful!
2023-07-29 08:38:33,612 - log_utils - WARNING - Validation successful!
2023-07-29 08:38:35,226 - log_utils - ERROR - An error occurred in read_kafka method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:38:35,226 - log_utils - ERROR - An error occurred in read_kafka method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:38:35,226 - log_utils - ERROR - An error occurred in read_kafka method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:38:35,239 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 42, in main
    deserialized_df = deserialize_df(schema, df, logger)
NameError: name 'deserialize_df' is not defined
2023-07-29 08:38:35,239 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 42, in main
    deserialized_df = deserialize_df(schema, df, logger)
NameError: name 'deserialize_df' is not defined
2023-07-29 08:38:35,239 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 42, in main
    deserialized_df = deserialize_df(schema, df, logger)
NameError: name 'deserialize_df' is not defined
2023-07-29 08:41:49,348 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:41:49,348 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:41:49,348 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:41:49,348 - log_utils - INFO - Started main() method...
2023-07-29 08:41:49,348 - log_utils - INFO - Started main() method...
2023-07-29 08:41:49,348 - log_utils - INFO - Started main() method...
2023-07-29 08:41:49,349 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:41:49,349 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:41:49,349 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:41:49,349 - log_utils - INFO - Running on dev environment
2023-07-29 08:41:49,349 - log_utils - INFO - Running on dev environment
2023-07-29 08:41:49,349 - log_utils - INFO - Running on dev environment
2023-07-29 08:41:51,347 - log_utils - INFO - Spark object created..
2023-07-29 08:41:51,347 - log_utils - INFO - Spark object created..
2023-07-29 08:41:51,347 - log_utils - INFO - Spark object created..
2023-07-29 08:41:51,347 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:41:51,347 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:41:51,347 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:41:51,347 - log_utils - WARNING - Validation successful!
2023-07-29 08:41:51,347 - log_utils - WARNING - Validation successful!
2023-07-29 08:41:51,347 - log_utils - WARNING - Validation successful!
2023-07-29 08:41:52,421 - log_utils - ERROR - An error occurred in read_stream method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:41:52,421 - log_utils - ERROR - An error occurred in read_stream method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:41:52,421 - log_utils - ERROR - An error occurred in read_stream method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:41:52,433 - log_utils - ERROR - An error occurred in deserialize_df method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 32, in deserialize_df
    deserialized_df = (df.select
AttributeError: 'NoneType' object has no attribute 'select'
2023-07-29 08:41:52,433 - log_utils - ERROR - An error occurred in deserialize_df method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 32, in deserialize_df
    deserialized_df = (df.select
AttributeError: 'NoneType' object has no attribute 'select'
2023-07-29 08:41:52,433 - log_utils - ERROR - An error occurred in deserialize_df method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 32, in deserialize_df
    deserialized_df = (df.select
AttributeError: 'NoneType' object has no attribute 'select'
2023-07-29 08:41:52,434 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 99, in main
    transformed_df = transformed_df(deserialized_df, logger)
UnboundLocalError: local variable 'transformed_df' referenced before assignment
2023-07-29 08:41:52,434 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 99, in main
    transformed_df = transformed_df(deserialized_df, logger)
UnboundLocalError: local variable 'transformed_df' referenced before assignment
2023-07-29 08:41:52,434 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 99, in main
    transformed_df = transformed_df(deserialized_df, logger)
UnboundLocalError: local variable 'transformed_df' referenced before assignment
2023-07-29 08:47:45,841 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:47:45,841 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:47:45,841 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:47:45,841 - log_utils - INFO - Started main() method...
2023-07-29 08:47:45,841 - log_utils - INFO - Started main() method...
2023-07-29 08:47:45,841 - log_utils - INFO - Started main() method...
2023-07-29 08:47:45,842 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:47:45,842 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:47:45,842 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:47:45,842 - log_utils - INFO - Running on dev environment
2023-07-29 08:47:45,842 - log_utils - INFO - Running on dev environment
2023-07-29 08:47:45,842 - log_utils - INFO - Running on dev environment
2023-07-29 08:47:47,908 - log_utils - INFO - Spark object created..
2023-07-29 08:47:47,908 - log_utils - INFO - Spark object created..
2023-07-29 08:47:47,908 - log_utils - INFO - Spark object created..
2023-07-29 08:47:47,908 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:47:47,908 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:47:47,908 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:47:47,908 - log_utils - WARNING - Validation successful!
2023-07-29 08:47:47,908 - log_utils - WARNING - Validation successful!
2023-07-29 08:47:47,908 - log_utils - WARNING - Validation successful!
2023-07-29 08:47:48,966 - log_utils - ERROR - An error occurred in read_stream method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:47:48,966 - log_utils - ERROR - An error occurred in read_stream method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:47:48,966 - log_utils - ERROR - An error occurred in read_stream method..
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 22, in read_stream
    .load()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 469, in load
    return self._df(self._jreader.load())
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 196, in deco
    raise converted from None
pyspark.sql.utils.AnalysisException:  Failed to find data source: kafka. Please deploy the application as per the deployment section of "Structured Streaming + Kafka Integration Guide".        
2023-07-29 08:47:48,978 - log_utils - INFO - Application done!!!
2023-07-29 08:47:48,978 - log_utils - INFO - Application done!!!
2023-07-29 08:47:48,978 - log_utils - INFO - Application done!!!
2023-07-29 08:56:16,620 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:56:16,620 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:56:16,620 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 08:56:16,620 - log_utils - INFO - Started main() method...
2023-07-29 08:56:16,620 - log_utils - INFO - Started main() method...
2023-07-29 08:56:16,620 - log_utils - INFO - Started main() method...
2023-07-29 08:56:16,621 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:56:16,621 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:56:16,621 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 08:56:16,621 - log_utils - INFO - Running on dev environment
2023-07-29 08:56:16,621 - log_utils - INFO - Running on dev environment
2023-07-29 08:56:16,621 - log_utils - INFO - Running on dev environment
2023-07-29 08:56:20,745 - log_utils - INFO - Spark object created..
2023-07-29 08:56:20,745 - log_utils - INFO - Spark object created..
2023-07-29 08:56:20,745 - log_utils - INFO - Spark object created..
2023-07-29 08:56:20,745 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:56:20,745 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:56:20,745 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 08:56:20,745 - log_utils - WARNING - Validation successful!
2023-07-29 08:56:20,745 - log_utils - WARNING - Validation successful!
2023-07-29 08:56:20,745 - log_utils - WARNING - Validation successful!
2023-07-29 08:56:24,288 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 99, in main
    transformed_df = transformed_df(deserialized_df, logger)
UnboundLocalError: local variable 'transformed_df' referenced before assignment
2023-07-29 08:56:24,288 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 99, in main
    transformed_df = transformed_df(deserialized_df, logger)
UnboundLocalError: local variable 'transformed_df' referenced before assignment
2023-07-29 08:56:24,288 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 99, in main
    transformed_df = transformed_df(deserialized_df, logger)
UnboundLocalError: local variable 'transformed_df' referenced before assignment
2023-07-29 09:01:51,831 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 09:01:51,831 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 09:01:51,831 - log_utils - INFO - logger object is created and file location is local dir
2023-07-29 09:01:51,831 - log_utils - INFO - Started main() method...
2023-07-29 09:01:51,831 - log_utils - INFO - Started main() method...
2023-07-29 09:01:51,831 - log_utils - INFO - Started main() method...
2023-07-29 09:01:51,833 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:01:51,833 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:01:51,833 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:01:51,833 - log_utils - INFO - Running on dev environment
2023-07-29 09:01:51,833 - log_utils - INFO - Running on dev environment
2023-07-29 09:01:51,833 - log_utils - INFO - Running on dev environment
2023-07-29 09:01:55,207 - log_utils - INFO - Spark object created..
2023-07-29 09:01:55,207 - log_utils - INFO - Spark object created..
2023-07-29 09:01:55,207 - log_utils - INFO - Spark object created..
2023-07-29 09:01:55,207 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 09:01:55,207 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 09:01:55,207 - log_utils - WARNING - Started the test_spark_obj method
2023-07-29 09:01:55,207 - log_utils - WARNING - Validation successful!
2023-07-29 09:01:55,207 - log_utils - WARNING - Validation successful!
2023-07-29 09:01:55,207 - log_utils - WARNING - Validation successful!
2023-07-29 09:03:56,539 - log_utils - ERROR - An error occurred in write_to_console_local() method:
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 81, in write_to_console_local
    .awaitTermination()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 107, in awaitTermination
    return self._jsq.awaitTermination()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 190, in deco
    return f(*a, **kw)
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/protocol.py", line 334, in get_return_value
    raise Py4JError(
py4j.protocol.Py4JError: An error occurred while calling o53.awaitTermination
2023-07-29 09:03:56,539 - log_utils - ERROR - An error occurred in write_to_console_local() method:
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 81, in write_to_console_local
    .awaitTermination()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 107, in awaitTermination
    return self._jsq.awaitTermination()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 190, in deco
    return f(*a, **kw)
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/protocol.py", line 334, in get_return_value
    raise Py4JError(
py4j.protocol.Py4JError: An error occurred while calling o53.awaitTermination
2023-07-29 09:03:56,539 - log_utils - ERROR - An error occurred in write_to_console_local() method:
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 81, in write_to_console_local
    .awaitTermination()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/streaming.py", line 107, in awaitTermination
    return self._jsq.awaitTermination()
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/java_gateway.py", line 1321, in __call__
    return_value = get_return_value(
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/pyspark.zip/pyspark/sql/utils.py", line 190, in deco
    return f(*a, **kw)
  File "/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/python/lib/py4j-0.10.9.5-src.zip/py4j/protocol.py", line 334, in get_return_value
    raise Py4JError(
py4j.protocol.Py4JError: An error occurred while calling o53.awaitTermination
2023-07-29 09:03:56,543 - log_utils - INFO - Application done!!!
2023-07-29 09:03:56,543 - log_utils - INFO - Application done!!!
2023-07-29 09:03:56,543 - log_utils - INFO - Application done!!!
2023-07-29 09:10:41,735 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 113, in main
    logger = setup_logger(log_level=prod_log_level, log_file=s3_logs)
  File "/home/ubuntu/aws_restbus_proj/source/log_utils.py", line 22, in setup_logger
    file_handler = logging.FileHandler(log_file)
  File "/usr/lib/python3.10/logging/__init__.py", line 1169, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.10/logging/__init__.py", line 1201, in _open
    return open_func(self.baseFilename, self.mode,
FileNotFoundError: [Errno 2] No such file or directory: '/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/bin/s3:/restbus-logs'
2023-07-29 09:10:41,735 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 113, in main
    logger = setup_logger(log_level=prod_log_level, log_file=s3_logs)
  File "/home/ubuntu/aws_restbus_proj/source/log_utils.py", line 22, in setup_logger
    file_handler = logging.FileHandler(log_file)
  File "/usr/lib/python3.10/logging/__init__.py", line 1169, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.10/logging/__init__.py", line 1201, in _open
    return open_func(self.baseFilename, self.mode,
FileNotFoundError: [Errno 2] No such file or directory: '/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/bin/s3:/restbus-logs'
2023-07-29 09:11:24,490 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 113, in main
    logger = setup_logger(log_level=prod_log_level, log_file=s3_logs)
  File "/home/ubuntu/aws_restbus_proj/source/log_utils.py", line 22, in setup_logger
    file_handler = logging.FileHandler(log_file)
  File "/usr/lib/python3.10/logging/__init__.py", line 1169, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.10/logging/__init__.py", line 1201, in _open
    return open_func(self.baseFilename, self.mode,
FileNotFoundError: [Errno 2] No such file or directory: '/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/bin/s3:/restbus-logs'
2023-07-29 09:11:24,490 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 113, in main
    logger = setup_logger(log_level=prod_log_level, log_file=s3_logs)
  File "/home/ubuntu/aws_restbus_proj/source/log_utils.py", line 22, in setup_logger
    file_handler = logging.FileHandler(log_file)
  File "/usr/lib/python3.10/logging/__init__.py", line 1169, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.10/logging/__init__.py", line 1201, in _open
    return open_func(self.baseFilename, self.mode,
FileNotFoundError: [Errno 2] No such file or directory: '/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/bin/s3:/restbus-logs'
2023-07-29 09:13:35,765 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 113, in main
    logger = setup_logger(log_level=prod_log_level, log_file=s3_logs)
  File "/home/ubuntu/aws_restbus_proj/source/log_utils.py", line 22, in setup_logger
    file_handler = logging.FileHandler(log_file)
  File "/usr/lib/python3.10/logging/__init__.py", line 1169, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.10/logging/__init__.py", line 1201, in _open
    return open_func(self.baseFilename, self.mode,
FileNotFoundError: [Errno 2] No such file or directory: '/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/bin/s3:/restbus-logs'
2023-07-29 09:13:35,765 - log_utils - ERROR - An error occurred when calling main() method...
Traceback (most recent call last):
  File "/home/ubuntu/aws_restbus_proj/source/main.py", line 113, in main
    logger = setup_logger(log_level=prod_log_level, log_file=s3_logs)
  File "/home/ubuntu/aws_restbus_proj/source/log_utils.py", line 22, in setup_logger
    file_handler = logging.FileHandler(log_file)
  File "/usr/lib/python3.10/logging/__init__.py", line 1169, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.10/logging/__init__.py", line 1201, in _open
    return open_func(self.baseFilename, self.mode,
FileNotFoundError: [Errno 2] No such file or directory: '/home/ubuntu/downloads/spark-3.3.2-bin-hadoop3/bin/s3:/restbus-logs'
2023-07-29 09:16:23,958 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:16:23,958 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:16:23,958 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:16:23,958 - log_utils - INFO - Started main() method...
2023-07-29 09:16:23,958 - log_utils - INFO - Started main() method...
2023-07-29 09:16:23,958 - log_utils - INFO - Started main() method...
2023-07-29 09:16:23,960 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:16:23,960 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:16:23,960 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:16:23,960 - log_utils - INFO - Running on prod environment
2023-07-29 09:16:23,960 - log_utils - INFO - Running on prod environment
2023-07-29 09:16:23,960 - log_utils - INFO - Running on prod environment
2023-07-29 09:18:18,522 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:18:18,522 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:18:18,522 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:18:18,524 - log_utils - INFO - Started main() method...
2023-07-29 09:18:18,524 - log_utils - INFO - Started main() method...
2023-07-29 09:18:18,524 - log_utils - INFO - Started main() method...
2023-07-29 09:18:18,525 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:18:18,525 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:18:18,525 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:18:18,526 - log_utils - INFO - Running on prod environment
2023-07-29 09:18:18,526 - log_utils - INFO - Running on prod environment
2023-07-29 09:18:18,526 - log_utils - INFO - Running on prod environment
2023-07-29 09:20:36,003 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:20:36,003 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:20:36,003 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:20:36,003 - log_utils - INFO - Started main() method...
2023-07-29 09:20:36,003 - log_utils - INFO - Started main() method...
2023-07-29 09:20:36,003 - log_utils - INFO - Started main() method...
2023-07-29 09:20:36,005 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:20:36,005 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:20:36,005 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:20:36,005 - log_utils - INFO - Running on prod environment
2023-07-29 09:20:36,005 - log_utils - INFO - Running on prod environment
2023-07-29 09:20:36,005 - log_utils - INFO - Running on prod environment
2023-07-29 09:24:18,704 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:24:18,704 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:24:18,704 - log_utils - INFO - logger object is created and file location is s3
2023-07-29 09:24:18,704 - log_utils - INFO - Started main() method...
2023-07-29 09:24:18,704 - log_utils - INFO - Started main() method...
2023-07-29 09:24:18,704 - log_utils - INFO - Started main() method...
2023-07-29 09:24:18,705 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:24:18,705 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:24:18,705 - log_utils - INFO - Started get_spark_obj() method..
2023-07-29 09:24:18,706 - log_utils - INFO - Running on prod environment
2023-07-29 09:24:18,706 - log_utils - INFO - Running on prod environment
2023-07-29 09:24:18,706 - log_utils - INFO - Running on prod environment
